Feature: OpenACC implementation of momentum loop

drewsilcock opened the pull request.

drewsilcock committed:
Add naive OpenACC kernels to momentum u and v loops.

drewsilcock committed:
Update OpenACC flags to use CUDA compute capacity 70.

drewsilcock committed:
Ignore application output and nvvp profile files.

drewsilcock committed:
Move PRESENT check into two if statements in dl_timer.

The PGI compiler was seemingly moving the order of operations around inside the
if check, causing a seg fault.

drewsilcock committed:
Replace kernels with ACC parallel and loops.

AimyShaikh reviewed (approved):
All fine, good work Drew.

drewsilcock committed:
Add explicit OpenACC data directives for data transfer.

Still only momentum kernel that's being run on device, but constants
are all being copied over initially.
